### PR TITLE
Added a small note

### PR DESCRIPTION
**Pull request by IanStanion-google**
The note clarifies that the webRequestBlocking manifest key is replaced by declarativeNetRequest in MV3.

Addresses the need for this in these issues:
https://github.com/GoogleChrome/developer.chrome.com/issues/4286
https://github.com/GoogleChrome/developer.chrome.com/issues/2551

### CODE REVIEW

File: site/en/docs/extensions/reference/webRequest/index.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ As of Chrome 108, you can asynchronously supply credentials for [`onAuthRequired
 events](#event-onAuthRequired) if you use the `"webRequest"` and `"webRequestAuthProvider"`
 permissions.
 
+**note** As of Manifest V3, the `webRequestBlocking` manifest key is replaced by `declarativeNetRequest`, which enables use of the [declarativeNetRequest API](/docs/extensions/reference/declarativeNetRequest/).
```
Comment (oliverdunk) [inline site/en/docs/extensions/reference/webRequest/index.md:30]: A couple of thoughts:

- Policy installed extensions can still use `webRequestBlocking` (see the paragraph two above). Should we merge those two paragraphs to make it clear there are multiple options?
- Should we use an [aside](https://developer.chrome.com/docs/handbook/components/#asides) to format this?

Comment (jpmedley) [reply site/en/docs/extensions/reference/webRequest/index.md:30]: The answer to the last question is always 'yes'. Part of our job requirements is to implement corporate styles consistently across our site or sites.

**Comment by oliverdunk** [comment]
Hey @IanStanion-google, are you still working on this? I think given some of the discussion we've had internally it would be nice to write a longer paragraph covering all of the following:

- The `webRequestBlocking` permission is going away in MV3, and you should look at DNR.
- UNLESS your extension is installed by policy, in which case you can continue to use it.
- `webRequest` remains available and unchanged.

File: site/en/docs/extensions/reference/webRequest/index.md [hunk 2]
```diff
@@ -27,6 +27,8 @@ As of Chrome 108, you can asynchronously supply credentials for [`onAuthRequired
 events](#event-onAuthRequired) if you use the `"webRequest"` and `"webRequestAuthProvider"`
 permissions.
 
+**note** As of Manifest V3, the `webRequestBlocking` manifest key is replaced by `declarativeNetRequest`, which enables use of the [declarativeNetRequest API](/docs/extensions/reference/declarativeNetRequest/). Aside from `webRequestBlocking` , the webRequest API will be unchanged and available for normal use. Policy installed extensions will maintain their current use of `webRequestBlocking`.
```
Comment (oliverdunk) [inline site/en/docs/extensions/reference/webRequest/index.md:30]:
```suggestion
{% Aside %}
As of Manifest V3, the `webRequestBlocking` permission is no longer available for most extensions. Consider `declarativeNetRequest`, which enables use of the [declarativeNetRequest API](/docs/extensions/reference/declarativeNetRequest/). Aside from `webRequestBlocking`, the webRequest API will be unchanged and available for normal use. Policy installed extensions can continue to use `webRequestBlocking`.
{% endAside %}
```

**Review by oliverdunk — APPROVED** [review]
LGTM 👍

File: site/en/docs/extensions/reference/webRequest/index.md [hunk 3]
```diff
@@ -20,13 +20,14 @@ extension needs to have access to both the requested URL and its initiator. For
...
 
-If you are building policy-installed extensions for enterprises, and want to use the web request API
-in a blocking fashion, you need to request the `"webRequestBlocking"` permission.
-
 As of Chrome 108, you can asynchronously supply credentials for [`onAuthRequired`
 events](#event-onAuthRequired) if you use the `"webRequest"` and `"webRequestAuthProvider"`
 permissions.
 
+{% Aside %}
+As of Manifest V3, the `webRequestBlocking` permission is no longer available for most extensions. Consider `declarativeNetRequest`, which enables use of the [declarativeNetRequest API](/docs/extensions/reference/declarativeNetRequest/). Aside from `webRequestBlocking`, the webRequest API will be unchanged and available for normal use. Policy installed extensions can continue to use `webRequestBlocking`.
```
Comment (jpmedley) [inline site/en/docs/extensions/reference/webRequest/index.md:28]:
```suggestion
As of Manifest V3, the `"webRequestBlocking"` permission is no longer available for most extensions. Consider `"declarativeNetRequest"`, which enables use of the [declarativeNetRequest API](/docs/extensions/reference/declarativeNetRequest/). Aside from `"webRequestBlocking"`, the webRequest API will be unchanged and available for normal use. Policy installed extensions can continue to use `"webRequestBlocking"`.
```

Comment (jpmedley) [reply site/en/docs/extensions/reference/webRequest/index.md:28]: For anything in the manifest, quotes are part of the meaning of the identifier. That's why I always include them.

**Review by jpmedley — CHANGES_REQUESTED** [review]
Ian,

I'm sorry this escaped my attention. If I haven't gotten to something in 48 hours you should ping me about it.

Isn't the use of blocking web requests in enterprise only temporary? That should probably be mentioned.

**Comment by IanStanion-google** [comment]
@jpmedley Added the detail that policy installed are temporarily allowed to use webRequestBlocking. 
Let me know if you have any other change proposals!

**Review by jpmedley — APPROVED** [review]

**Comment by oliverdunk** [comment]
I'm slightly confused because to some extent these changes already seem to be live: https://developer.chrome.com/docs/extensions/reference/webRequest/. But happy with what's here.

**Review by oliverdunk — APPROVED** [review]

**Pull request merged**